parsec/core/fs/workspacefs/workspacefs.py

```diff
@@ -324,41 +324,26 @@ async def truncate(self, path: AnyPath, length: int) -> None:
         path = FsPath(path)
         await self.transactions.file_resize(path, length)
 
-    async def read_bytes(self, path: AnyPath, size: int = -1, offset: int = 0) -> bytes:
+    async def open_file(self, path: AnyPath, mode="rb"):
```

Comment:
no return typing ?

Reply:
@Sithi5 Could you fix the typing for `open_file` and `WorkspaceFile`?

Reply:
I did it for `open_file` and for `readable/writable/seekable` method of WorkspaceFile.